crates/matrix-sdk/src/client/mod.rs

```diff
@@ -2471,6 +2471,12 @@ impl Client {
         let base_room = self.inner.base_client.room_knocked(&response.room_id).await?;
         Ok(Room::new(self.clone(), base_room))
     }
+
+    /// Remove the room from the store, removing all data related to it.
+    pub async fn forget_room(&self, room_id: &RoomId) -> Result<()> {
+        self.inner.base_client.forget_room(room_id).await?;
+        Ok(())
+    }
```

Comment:
There's `Room::forget()`, and it would be easy to mistake one for the other. If you plan to keep this one, now you know there's `Room::forget()`: can you precise in the doc comment this does NOT forget the room at the server level, but only locally? (and as such, the room can still be in the DM list, etc.)

Reply:
Oh, I didn't realise this was already there. I'll remove it and add bindings to `Room::forget` instead.